modules/sbo-data-model.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// *openshift-docs/applications/connecting_applications_to_services/exposing-binding-data-from-a-service
+[id="sbo-data-model_{context}"]
+= Data model
+
+Service binding annotations follow the convention:
+
+[source,yaml]
+----
+service.binding(/<NAME>)?: "<VALUE>|(path=<JSONPATH_TEMPLATE>(,objectType=<OBJECT_TYPE>)?(,elementType=<ELEMENT_TYPE>)?(,sourceKey=<SOURCE_KEY>)?(,sourceValue=<SOURCE_VALUE>)?)"
+----
+where:
+[horizontal]
+`<NAME>`:: Specifies the name under which the binding value is going to be exposed. It can be excluded only when the `objectType` parameter is set to `Secret` or `ConfigMap`.
+`<VALUE>`:: Specifies the constant value exposed, when no `path` is set.
+
+This section explains the data model used in the annotations and OLM descriptors. The data model is the same for custom resource definition (CRD) or custom resource (CR) annotations and Operator Lifecycle Manager (OLM) descriptors, but the syntax is different, which is explained in the respective sections.
+
+The data model provides the details on the allowed values and semantic for the `path`, `elementType`, `objectType`, `sourceKey`, and `sourceValue` parameters.
```

Comment:
```suggestion
The data model provides the details on the allowed values and semantic for the `path`, `elementType`, `objectType`, `sourceKey`, and `sourceValue` parameters.
```
Not sure what `semantic` means here. It's usually used only as an adjective and not a noun. Should it be `syntax` instead?

Reply:
Nope, it is semantic here. It relates to the meaning of the phrases or logic that the user needs to know when using these parameters and their values. OCP docs use this word in this context. The OCP users, as cluster admins and backing service authors, are aware of this word, usage, and context.

Reply:
Thanks for the clarification @Srivaralakshmi. That makes sense. I've never seen that word used that way until now. I've learned something new, so thanks again for the explanation!